docs/sphinx/source/variables_style_rules.rst

```diff
@@ -0,0 +1,30 @@
+.. _variables_style_rules:
+
+Variables and Symbols
+=====================
+
+There is a convention on consistent variable names throughout the library:
+
+.. csv-table:: List of used Symbols and Variables
+   :file: ../../../pvlib/data/variables_style-rules.csv
+   :delim: ;
+   :header-rows: 1
+   :widths: 5, 5
+   :stub-columns: 1
+   
+For an definition and further help on symbols and units refer to the following sources:
+
+
+* `Reference Variable List by PVPMC <https://pvpmc.sandia.gov/resources/variable-list/>`_
+* with technical focus in `IEC 61724:1998 -- Photovoltaic system performance monitoring - Guidelines for measurement, data exchange and analysis <https://webstore.iec.ch/publication/5733>`_ ; the Indian Standard referencing the global IEC standard is available online: `IS/IEC 61724 (1998) <https://law.resource.org/pub/in/bis/S05/is.iec.61724.1998.pdf>`_
+* with physics focus:
+
+  * Explanation of Solar irradiation and solar geometry by `SoDa Service <http://www.soda-pro.com/home>`_ and partner pages (http://www.soda-is.com/, http://pro.soda-is.com/)
```

Comment:
Why do you include the "partner pages"? They both request that you just use soda-pro.com